src/plugins/discover/public/application/components/table/table.tsx

```diff
@@ -61,8 +62,6 @@ export const DocViewerTable = ({
     indexPattern?.fields,
   ]);
 
-  const [childParentFieldsMap] = useState({} as Record<string, string>);
```

Comment:
Nice cleanup BTW, wonder what this was for, seems state here was directly mutated.